### DISCUSSION TITLE
Account Suspended for no reason

### DISCUSSION THREAD

**Discussion by raymondfajiculay** [community/community#61056] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My account has been suspended(cuddfuzz), and all my projects were on that account. I was trying to clean my account, so I deleted some repositories and added new ones. I had planned to apply for a job tomorrow since I just graduated, but it seems that won't be possible now because all of my projects are in that account. I have nothing to show on my resume and portfolio. Can someone help me or provide information on why I might have been suspended? Also, how long does it take to get my account back

here is my ticket id: 2254545

**Comment by queenofcorgis** [comment, marked as answer]
Hi @conefuzed,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by 0716690821** [comment]
My account has been suspended(cuddfuzz), and all my projects were on that account. I was trying to clean my account, so I deleted some repositories and added new ones.
